website/docs/atmo/usage/creating-runnables.md

```diff
@@ -10,7 +10,19 @@ You can create a new Runnable with subo:
 subo create runnable myfunction
 ```
 
-By default, Rust will be used. To use JavaScript, pass `--lang`:
+By default, Rust will be used. To use one of the other [supported languages](https://docs.suborbital.dev/reactr/language-support), pass the relevant value of the `--lang` parameter:
```

Comment:
Internal links should reference the `.md` file, so that the build configuration is properly respected.

As you can see in the [preview](https://suborbital.github.io/docs/laura/22-list-supported-languages-for-runnables/atmo/usage/creating-runnables) this absolute link will end up pointing outside of the preview, instead of the relevant page of the preview.

Reply:
Fixed!